opencloudmesh/lib/ShareeSearchPlugin.php

```diff
@@ -124,14 +124,7 @@ public function search($search) {
 							'shareType' => Share::SHARE_TYPE_REMOTE,
 							'shareWith' => $cloudId,
 							'server' => $serverUrl,
-						],
-						'label' =>  $contact['FN'],
-						'value' => [
-							'shareType' => Share::SHARE_TYPE_REMOTE_GROUP,
-							'shareWith' => $cloudId,
-							'server' => $serverUrl,
-						],
-					];
+						]];
```

Comment:
Why remove these lines?

Reply:
because when you find the searched phrase in some contacts metadata it should be a user (person), not a group.
it is unusual to save a group as a contact. right?

Reply:
What if there is a user called scientist AND there is a group called scientist?
Kindly, please do not mark a conversation as resolved while I have not responded yet. :-)

Reply:
just scientist? The scientist is a local user/group and the shareeSearchPlugin is in the circuit when the search has some @ to indicate the user is remote.

Reply:
I will describe the workflow to explain the system: 
1- If you type a search phrase without any @ the code will jump from running any logic and return an empty array. 
2- if you send a Search phrase that contains at least one @:
   2-1- code search in all contacts that have the same CloudId `CLOUD` in their Contact. 
   2-2- code search in all contacts that have the same other property like `FN` in their Contact. 
   2-2- if the above Items have no result to provide code will generate two results. first one for USER and second one For group.